NDB.Covid19/NDB.Covid19.Droid/Resources/layout/welcome_what_is_new.xml

```diff
@@ -1,81 +1,72 @@
 <androidx.constraintlayout.widget.ConstraintLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent">
+	xmlns:android="http://schemas.android.com/apk/res/android"
+	xmlns:app="http://schemas.android.com/apk/res-auto"
+	android:layout_width="match_parent"
+	android:layout_height="match_parent"
+	android:paddingStart="@dimen/defaultHorizontalMargin"
+	android:paddingEnd="@dimen/defaultHorizontalMargin">
 
-    <TextView
-        android:id="@+id/welcome_what_is_new_title"
-        style="@style/HeaderText"
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        android:layout_alignParentTop="true"
-        android:paddingTop="44dp"
-        android:paddingHorizontal="25dp"
-        android:text="Nyt smitte|stop"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+	<TextView
+		android:id="@+id/welcome_what_is_new_title"
+		style="@style/HeaderText"
+		android:layout_width="0dp"
+		android:layout_height="wrap_content"
+		android:layout_alignParentTop="true"
+		android:paddingTop="@dimen/defaultHorizontalMargin"
+		android:text="What is new in Smittestopp"
+		app:layout_constraintEnd_toEndOf="parent"
+		app:layout_constraintStart_toStartOf="parent"
+		app:layout_constraintTop_toTopOf="parent" />
 
-    <ScrollView
-        android:id="@+id/scroll_view"
-        style="@style/ScrollScreen"
-        android:layout_width="match_parent"
-        android:layout_height="0dp"
-        app:layout_constraintBottom_toTopOf="@id/ok_button"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/welcome_what_is_new_title"
-        android:fillViewport="true">
+	<ScrollView
+		android:id="@+id/scroll_view"
+		style="@style/ScrollScreen"
+		android:layout_width="match_parent"
+		android:layout_height="0dp"
+		app:layout_constraintEnd_toEndOf="parent"
```

Comment:
are you sure it will behave correctly without bottom constraint?

Reply:
i mean it can overlap on the button if the text will be long for big font sizes

Reply:
i guess so

Reply:
try to write long dummy text and change display size and text size to the largest